src/utils/overlayFS.ts

```diff
@@ -13,6 +13,9 @@ export interface OverlayMergedFS extends AsyncDisposable {
 
 export interface DisposableOverlayBaseFS extends OverlayBaseFS, AsyncDisposable {}
 
+// @ts-ignore
+Symbol.asyncDispose ??= Symbol("Symbol.asyncDispose");
```

Comment:
What version of node are you on? This should be present.

Reply:
v20.12.1

Reply:
Hm, was this crashing locally or in CI? 

![image](https://github.com/microsoft/typescript-error-deltas/assets/5341706/1ea577d8-5e94-476f-9db8-0e05c0ac46de)


(It passed for me previously when I introduced this code, anyway....)

Reply:
Locally, I assume it would crash on CI but I can give it a try without it

Reply:
Wow, it does fail. That doesn't make any sense at all...

Reply:
It fails on CI as well

Reply:
Oh, this is because jest makes an entirely different environment that's missing these, even though the system has it: https://github.com/jestjs/jest/issues/14874

I guess we can leave it in, though we could probably also just swap jest out for vitest.

Reply:
I put it back then. We can track migrating to vitest on issue #153